google/cloud/bigquery/enums.py

```diff
@@ -292,6 +293,7 @@ class SqlTypeNames(str, enum.Enum):
     TIME = "TIME"
     DATETIME = "DATETIME"
     INTERVAL = "INTERVAL"  # NOTE: not available in legacy types
+    RANGE = "RANGE"
```

Comment:
is the comment for INTERVAL about legacy availability also true for RANGE?

Reply:
I guess the legacy types here refer to the [types supported by legacy SQL](https://cloud.google.com/bigquery/docs/data-types#legacy_sql_data_types)? In that case, it is indeed not supported. If so, I will add the comment here.